Network/Socket/Options.hsc

```diff
@@ -376,6 +389,60 @@ getSockOpt s (SockOpt level opt) = do
                 c_getsockopt fd level opt ptr ptr_sz
         peek ptr
 
+
+socketOptionPairs :: [Pair SocketOption String]
+socketOptionPairs =
+    [ (UnsupportedSocketOption, "Unsupported")
```

Comment:
I prefer to "UnsupportedSocketOption" so that the string is the same as the pattern synonym.